os/arch/arm/src/armv7-m/up_schedyield.c

```diff
@@ -73,6 +73,11 @@ void up_schedyield(void)
 		return;
 	}
 
+#ifdef CONFIG_TASK_SCHED_HISTORY
+	/* Save the task name which is yielding cpu resource */
```

Comment:
Please change the comments at all places. Let the comment be uniform indicating that we are logging the details of the task being switched out.